book/src/libs/wasp/benchspy/overview.md

```diff
@@ -0,0 +1,15 @@
+# BenchSpy
+
+BenchSpy (short for Benchmark Spy) is a [WASP](../overview.md)-coupled tool designed for easy comparison of various performance metrics.
```

Comment:
I'd underline "why you need it Bench Spy", e.g. find regressions between commits or releases, may be some other cases.